Dockerfile

```diff
@@ -2,18 +2,27 @@ FROM mambaorg/micromamba:1.2.0-bullseye-slim
 LABEL org.opencontainers.image.authors="us@couchbits.com"
 LABEL org.opencontainers.image.vendor="couchbits GmbH"
 
-COPY --chown=$MAMBA_USER:$MAMBA_USER environment.yml /tmp/env.yaml
-RUN micromamba install -y -n base -f /tmp/env.yaml && \
-    micromamba clean --all --yes
-
-# the app
+# the app location
 ENV PROJECT_DIR /moveapps-python-sdk
 WORKDIR $PROJECT_DIR
+
+# the OS depedencies
+USER root
+RUN chown -R $MAMBA_USER:$MAMBA_USER $PROJECT_DIR
+RUN apt-get update \
+    # fix for: `ImportError: libGL.so.1: cannot open shared object file: No such file or directory`
+    && apt-get install ffmpeg libsm6 libxext6  -y
```

Comment:
These OS libraries are needed to run the app on linux. It solves the error `ImportError: libGL.so.1: cannot open shared object file: No such file or directory`